core/src/commands/mention.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ICommand, ExecuteState, TextAreaTextApi } from './';
+import { selectWord, executeCommand } from '../utils/markdownUtils';
+
+export const mention: ICommand = {
+  name: 'mention',
```

Comment:
If you can't provide a way to get its dropdown, I'd recommend removing it and leaving it to user customization. @RARgames

Reply:
I can, in my project (I'm reworking it right now). However I'm not sure how it should work here as we don't know from what repo to get users.

Reply:
Maybe add it, but not include it in the default toolbar until we e.g. add some field to MDEditor so we can get repo/list of repos to get users from e.g.:

```js
<MDEditor
  mentionRepos={someReposList}
/>
```

Reply:
@RARgames  You don't need to add a new prop, but instead, you can extend the `commands` array with the `mention` command. Here's an example:

```jsx
 <MDEditor
    commands={[
      commands.mention(() => {
        return [...]
      })
    ]}
/>
```

```tsx
type Mention = (fn: any) => ICommand
export const mention: Mention = (fn) => {
  return {
    name: 'mention',
    //...
  }
}
```

You can omit the `mention` command if it's not fully functional.

Reply:
Mention command removed until fully implemented